nunit-project-editor.nuspec

```diff
@@ -0,0 +1,23 @@
+﻿<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>nunit-project-editor</id>
+    <version>1.0</version>
```

Comment:
@CharliePoole @ChrisMaddock this should work in exactly the same way as you do it for NuGet nuspec files. This is how we do it for Cake:

https://github.com/cake-build/cake/blob/develop/build.cake#L231
https://github.com/cake-build/cake/blob/develop/nuspec/Cake.Portable.nuspec#L7